### PR TITLE
Add features for VF-C installation

### PR DESCRIPTION
**Pull request by shank7485**

### CODE REVIEW

File: tests/test_vfc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```
Comment (electrocucaracha) [inline tests/test_vfc:2]: You have to source the source /var/onap/vfc folder

File: tests/test_vfc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+covered_functions=(
+"clone_all_vfc_repos"
+)
+
+# test_clone_all_vfc_repos() - Verify cloning and pulling source code from repositories
+function test_clone_all_vfc_repos {
+    # TODO(sshank)
+    echo ""
+}
```
Comment (electrocucaracha) [inline tests/test_vfc:14]: The functions needs to call the test functions directly like

https://github.com/electrocucaracha/vagrant-onap/blob/master/tests/test_mr#L20-L23

File: tests/test_vfc (hunk 1 above)
Comment (shank7485) [reply tests/test_vfc:2]: Done

File: tests/test_vfc (hunk 2 above)
Comment (shank7485) [reply tests/test_vfc:14]: Done

**Review by shank7485 — COMMENTED** [review]
Done

**Pull request merged**